chart/templates/NOTES.txt

```diff
@@ -17,7 +17,7 @@
     to monitor continuously.
 
     {{- if not .Values.features.eirini.enabled }}
-    {{- $cell_count := .Values.sizing.diego_cell.instances | default 1 }}
+    {{- $cell_count := .Values.sizing.diego_cell.instances | default (ternary 3 1 .Values.high_availability) }}
```

Comment:
I doubt this will ever change, but instead of hard-coding the `3` you should look it up in the manifest:

```
{{- $_ := include "_config.lookupManifest" (list $ "instance_groups/name=diego-cell/instances") }}
{{- $cell_count := .Values.sizing.diego_cell.instances | default (ternary $.kubecf.retval 1 .Values.high_availability) }}
```

Reply:
Ok. Will make a follow-up PR for this.